### PR TITLE
edits consistent with UI philosophy

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees PTAL

These are edits based on the feedback we got from the UI team in some other recent PRs.

### CODE REVIEW

**Comment by bparees** [comment]
@spadgett ptal

@gabemontero please tag spadgett in all these as he had the original suggestions for reworking them (I think he had a comment about the use of periods in the descriptions also)

File: openshift/templates/django.json [hunk 1]
```diff
@@ -270,27 +269,24 @@
...
       "name": "DATABASE_USER",
-      "displayName": "Database user name",
-      "description": "Database user name"
+      "displayName": "Database Username"
     },
     {
       "name": "DATABASE_PASSWORD",
-      "displayName": "Database user password",
-      "description": "Database user password"
+      "displayName": "Database User password"
```
Comment (spadgett) [inline openshift/templates/django.json]: Database User Password

File: openshift/templates/django-postgresql.json [hunk 2]
```diff
@@ -339,84 +339,80 @@
...
     {
       "name": "APP_CONFIG",
-      "displayName": "Application configuration file path",
+      "displayName": "Application Configuration File Path",
       "description": "Relative path to Gunicorn configuration file (optional)"
     },
     {
       "name": "DJANGO_SECRET_KEY",
-      "displayName": "Djange secret key",
+      "displayName": "Djange Secret Key",
```
Comment (spadgett) [inline openshift/templates/django-postgresql.json]: Should this be "Django Secret Key"?

File: openshift/templates/django.json [hunk 3]
```diff
@@ -270,27 +269,24 @@
...
     {
       "name": "APP_CONFIG",
-      "displayName": "Application configuration file path",
+      "displayName": "Application Configuration File Path",
       "description": "Relative path to Gunicorn configuration file (optional)"
     },
     {
       "name": "DJANGO_SECRET_KEY",
-      "displayName": "Djange secret key",
+      "displayName": "Djange Secret Key",
```
Comment (spadgett) [inline openshift/templates/django.json]: Django?

**Comment by spadgett** [comment]
Same comments here as in https://github.com/openshift/dancer-ex/pull/34

File: openshift/templates/django-postgresql.json [hunk 4]
```diff
@@ -339,84 +339,80 @@
...
-      "description": "Database service name",
+      "displayName": "Database Service Name",
       "value": "postgresql"
     },
     {
       "name": "DATABASE_ENGINE",
-      "displayName": "Database engine",
+      "displayName": "Database Engine",
       "description": "Database engine: postgresql, mysql or sqlite (default)",
       "value": "postgresql"
```
Comment (spadgett) [inline openshift/templates/django-postgresql.json]: Not new to your change, but it's odd that value is set to something other than what the description says is the default.

**Comment by gabemontero** [comment]
edits from comments from @spadgett pushed

**Comment by spadgett** [comment]
LGTM

**Comment by bparees** [comment]
lgtm

**Pull request merged**